### ISSUE TITLE
Line breaks for static text inside 'select' are ignored

### ISSUE THREAD

**Issue by vtoms4** [jeffijoe/messageformat.net#34]
Library version: `6.2`.
Sample message:
```
"Single text which will not change.\nSummary:{acceptedData, select, NONE {} other {\nAccepted Data:{acceptedData}}}"
```
Message arguments:
```
{
    "acceptedData": "\n-X\n-Y\n-Z"
}
```
MessageFormatter usage:
```
var messageFormatProvider = new MessageFormatter(true, "en");
var result = messageFormatProvider.FormatMessage(message, messageArguments)
```
Current result:
```
"Single text which will not change.\nSummary:Accepted Data:\n-X\n-Y\n-Z"
```
Current result from VisualStudio Text Visualizer:
![image](https://github.com/jeffijoe/messageformat.net/assets/142882994/58a24a07-1a30-4940-9241-fdbd9d576964)
Expected result:
```
"Single text which will not change.\nSummary:\nAccepted Data:\n-X\n-Y\n-Z"
```

**Comment by jeffijoe** [comment]
So the first newline in the `other` branch is being omitted?

**Comment by vtoms4** [comment]
Yes. And it could be anywhere in this static text. Also `other {Accepted\nData:{acceptedData}}` will not keep the newline between 'Accepted' and 'Data'.

**Comment by jeffijoe** [comment]
Interesting, that definitely sounds like a bug. I'll look into it.

**Comment by jeffijoe** [comment]
Fixed and released as 7.0.0, thanks for reporting!
